Makefile

```
@@ -0,0 +1,15 @@
+.PHONY: tests setup mocks clean list build_clean_script
+
+list:
+	@echo "\n==========================="
+	@echo "==    Swifty Scripty    =="
+	@echo "==========================="
+	@echo "==     Commands list     =="
+	@echo "==========================="
+	@echo "build_scripts"
+
+build_scripts:
+	@swift package clean
+	@swiftyscripty -s
+	@swift build --configuration release
+	@cp -f .build/release/SwiftyScriptyCLI ./SwiftyScriptyCLI
```

```suggestion
	@cp -f .build/release/SwiftyScriptyCLI ./swiftyscripty
```
Che dici? almeno è più semplice poi copiarla nella cartella corretta

---

non è necessario tanto abbiamo il comando `make cli`